Add desktop version

masangana opened the pull request:
- Adding css for desktop version
- Adding ressource for desktop version

masangana committed:
Fix css code

fixing css code for image
remove useless css code

masangana committed:
Fix linters errors

Fixing linters errors

zieeco commented on style.css:
```diff
@@ -639,7 +638,7 @@ footer {
+    width: 620px;
   }
 
   .works-section-project-description {
```
- [ ] Hey, Great job on your design. Please, kindly reduce the content of this paragraph to take up a maximum of  5 lines.
:point_right:     ![button](https://user-images.githubusercontent.com/59397159/165629200-564aaf58-be2c-4759-99dd-9ce7c1572c68.png)

- [ ] Please, use the text content of the paragraph from Figma so that your text  takes a maximum of 3 lines below:

**YOUR DESIGN** :point_down:      ![para2](https://user-images.githubusercontent.com/59397159/165629184-199f71d7-612e-4cbd-bd43-c9638e0a15a2.png)

**FIGMA DESIGN** :point_down: 
![para](https://user-images.githubusercontent.com/59397159/165629192-5bf4c725-f54e-47bb-befb-67f8f015ffe4.png)

- [ ] Kindly adjust the margin so your cards are well aligned below:


> || **YOUR DESIGN** || FIGMA DESIGN ||
> |-|---------|-|---------|-|
> || ![yeah](https://user-images.githubusercontent.com/59397159/165634341-a0a6c012-c918-4fff-8125-1e73c6494df6.png) || ![yeahh](https://user-images.githubusercontent.com/59397159/165634583-8e9065e9-09c6-4ee9-808c-a9e689825646.png)  || 

- [x] Kindly, increase the width of your about text to make it grow almost to the middle below:


> || **FROM THIS** || TO THIS||
> |-|---------|-|---------|-|
> || ![enter3](https://user-images.githubusercontent.com/59397159/165635569-a06bc0df-20f2-4087-bcca-f962e2ea591a.png) || ![center](https://user-images.githubusercontent.com/59397159/165635688-42a00bbd-fdf0-4388-8ab3-094bdea63f7d.png) ||

masangana replied:
I did the changes but for the last, i think what i did is most near the template that your idea

zieeco commented on style.css:
```diff
@@ -670,7 +669,6 @@ footer {
   }
 
   .project-button {
-    /*display: none;*/
     margin-bottom: 0;
   }
```
- [ ] Please, kindly add the link to the Github repository or the Live demo link  when a user clicks on this button.
:point_down: 
![button](https://user-images.githubusercontent.com/59397159/165627382-8874f00d-f237-4de9-9c47-456991a381c3.png)

masangana replied:
i added the link to the github repo

zieeco reviewed (changes requested):
Hi @masangana 


Great job on this project👏 
You've done well implementing this design however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/a
**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

masangana committed:
Fix css and html to match with the template

--realign the box (recent works)
--reduce text to match with the template
--realign box project
